### PR TITLE
Config file and its test created

### PR DESCRIPTION
**Pull request by Boreas09**

### CODE REVIEW

**Comment by ermvrs** [comment]
Improve tests & file reading. For example throw an error if file is not exist or not a valid JSON. Read file and parse the json then assign this js object into variable in configReader. So we won't need to read file everytime

File: src/utils/configReader.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  let config: Config
+  try {
+    config = JSON.parse(rawData)
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  } catch (error: any) {
+    throw new Error(`Error parsing JSON configuration file: ${error.message}`)
+  }
+
+  // Check if the required properties exist and have the correct types
+  if (typeof config.appName !== 'string') {
```
Comment (ermvrs) [inline src/utils/configReader.ts:40]: We can make these checks more clear. Define an array of property names and loop through to check that all necessary values are exist on config.

**Review by ermvrs — CHANGES_REQUESTED** [review]

File: tests/utils/configReader.test.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import config from '../../src/utils/configReader'
+
+describe('Config file check', () => {
```
Comment (ermvrs) [inline tests/utils/configReader.test.ts:3]: Add edge cases, if config file not json, not exist etc.

**Review by ermvrs — CHANGES_REQUESTED** [review]
